component/_buttons.scss

```diff
@@ -1,57 +1,82 @@
 // @file buttons.scss
 
-$form-button-background: palette(brand);
-$form-button-border: false;
-$form-button-border-color: palette(brand, dark);
+// Remove and edit styles as needed. This basic idea provides three button styles
+// Can easily be expanded to accomodate more
+
+$form-button-background-default: palette(mono, 10);
+$form-button-color-default: palette(mono, 66);
+$form-button-border-color-default: palette(mono, 33);
 $form-button-border-width: 1px;
 $form-button-border-radius: $default-border-radius;
-$form-button-padding: .325em 1em .25em;
-
-// Default button style. Classes must be chained to change colors, i.e. class="button button--primary"
-a.button,
-button.button,
-input.button {
-	@include appearance(button);
-	@include transition( all 250ms ease );
-	border-radius: $form-button-border-radius;
-	cursor: pointer;
-	margin-bottom: .5em;
-	padding: $form-button-padding;
+$form-button-padding: .625em 1em .5em;
+
+//
+// Usage: @include button()
+// Accepts: null (default), primary, secondary
+// Returns: A compiled set of buttons styles including idle and hover
+// Authors: The Oomph Team
+//
+@mixin button($type:default) {
+  border: $form-button-border-width solid $form-button-border-color-default;
+  border-radius: $form-button-border-radius;
+  display: inline-block;
+  padding: $form-button-padding;
+  text-transform: uppercase;
+
```

Comment:
No, you are not way off. I asked in the UX room about this and got a lukewarm response. Here is the thing about a Scaffold… we are always walking the line of having enough in here to get started from a good place, or, stripping it bare and having to rewrite the same things over and over again when we start a project. I'd like to leave them in because it sets up the expectation that we should be a DRY as can be. They are there to use for the project, even though they may not be used for _this_ scaffold example. 

And I usually want my input fields and buttons to be the same height, therefore, the same font size and padding for each. These buttons ship with the same size across the board no matter the type, and a similar size to the input fields. 

<img width="875" alt="screen shot 2016-12-14 at 7 50 09 am" src="https://cloud.githubusercontent.com/assets/1530232/21182617/fc09fb00-c1d1-11e6-834a-a42691f15ad5.png">